endpoints/openrtb2/auction.go

```diff
@@ -1931,6 +1965,96 @@ func getAccountID(pub *openrtb2.Publisher) string {
 	return metrics.PublisherUnknown
 }
 
+// Returns the account from the request
+func getAccountIdFromRawRequest(hasStoredRequest bool, storedRequest json.RawMessage, originalRequest []byte) (string, bool, []error) {
+	var accountId string
+	var isAppReq bool
+	var err error
+
+	if hasStoredRequest {
+		accountId, isAppReq, err = searchAccountId(storedRequest)
+		if err != nil {
+			return "", isAppReq, []error{err}
+		}
+	} else {
+		accountId, isAppReq, err = searchAccountId(originalRequest)
+		if err != nil {
+			return "", isAppReq, []error{err}
+		}
+	}
```

Comment:
Can be simplified a little

```suggestion
        request := originalRequest
        if hasStoredRequest {
                request = storedRequest
        }
        
	accountId, isAppReq, err = searchAccountId(request)
	if err != nil {
		return "", isAppReq, []error{err}
	}
```